test/jdk/javax/swing/JMenu/TestUngrab.java

```diff
@@ -27,7 +27,6 @@
  * @test
  * @bug 8267374
```

Comment:
Should the current bug id be updated here?

Reply:
No, we only put the product ids in the tag and this is test-only fix

Reply:
Ok.